### PR TITLE
Refactor knexfile and migrations

### LINKED ISSUE #17: Create migration files using Knex

### PR DESCRIPTION
**Pull request by cagallo**
## What Features are Being Added?

- Added single migration file for books and favorites 

## What is Being Refactored?

- deleted previous migration files (separate files for books and favorites)
- changed database name to 'book-club'
- changed column names to snake_case

## Where should the reviewer start?

- knexFile.js

## How should this be manually tested?

pgAdmin4 or `psql postgres` on command line

## What are the relevant tickets?

- Create database

## Any Other Notes/Questions/Problems?

- None, let me know if you have any questions!

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -8,16 +8,16 @@ module.exports = {
   development: {
     client: 'postgresql',
     connection: {
-      database: 'books',
-      user: 'postgres',
-      password: 'Booger9s'
+      database: 'book_club',
```
Comment (krdurrett) [inline knexfile.js:11]: good idea to rename db to book_club

File: migrations/20220219215644_create_books_favorites.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
```
Comment (krdurrett) [inline migrations/20220219215644_create_books_favorites.js:5]: This function to create both tables looks so much better!

File: migrations/20220219215644_create_books_favorites.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  return knex.schema.createTable('books', (table) => {
```
Comment (Fordo29) [inline migrations/20220219215644_create_books_favorites.js:6]: Nice way to combine the creation of both sets of data tables in the same function

**Review by krdurrett — APPROVED** [review]
Great work refactoring the migration to include both tables

File: migrations/20220219215644_create_books_favorites.js (hunk 2 above)
Comment (ectweitmann) [reply migrations/20220219215644_create_books_favorites.js:5]: Yeah I've learned so much from all the errors we got lol

**Pull request merged**